roles/confluent.test/molecule/archive-scram-rhel/verify.yml

```diff
@@ -89,7 +89,7 @@
       vars:
         file_path: /opt/confluent/etc/kafka/connect-distributed.properties
         property: plugin.path
-        expected_value: /opt/confluent/confluent-6.0.2/share/java,/usr/share/java
+        expected_value: /opt/confluent/confluent-6.0.4/share/java,/usr/share/java
```

Comment:
For 6.0.5 release, should we change it to 6.0.5 or would this be fine?

Reply:
It should be changed to 6.0.5

Reply:
I'm assuming bumpversion will take care of this.

Reply:
This will be done by bumpversion script. I have just brought it to latest before running the script i.e. 6.0.4 and added this file in .bump2version.cfg